src/components/MessagesList/MessagesGroup/Message/MessagePart/DeckCard.vue

```diff
@@ -141,6 +141,7 @@ export default {
 
 .wide {
 	max-width: 400px;
+	width: 100%;
```

Comment:
Can you check it looks okay in the chat list?

Reply:
Top: Desktop Client
Bottom: Web (current 27.1.0)
![image](https://github.com/nextcloud/spreed/assets/93392545/9bc50bd9-a9ba-4a52-8b5e-689494ed9dde)